Cargo.toml

```diff
@@ -24,11 +24,6 @@ doc = false
 name = "ldap-check"
 path = "src/ldap-check.rs"
 
-[[bin]]
-doc = false
-name = "migrate-db"
```

Comment:
this was to transition the old json format into the current sqlite database format. could have removed it awhile ago, and now it fails to compile. 

🔥